flocker/testtools/_base.py

```diff
@@ -11,7 +11,20 @@
 import testtools
 from testtools.content import text_content
 from testtools.deferredruntest import (
-    AsynchronousDeferredRunTestForBrokenTwisted)
+    AsynchronousDeferredRunTestForBrokenTwisted,
+)
+
+try:
+    from testtools.deferredruntest import CaptureTwistedLogs
+except ImportError:
+    # We are using a fork of testtools, which unfortunately means that we need
+    # to do special things to make sure we're using the latest version. Raise
+    # an error message that will help people figure out what they need to do.
+    raise Exception(
+        'Cannot import CaptureTwistedLogs. Maybe upgrade your version of '
+        'testtools: pip install --upgrade --process-dependency-links .[dev]'
```

Comment:
:grey_question: Optional for this PR, but commenting anyway because you added this Exception.

doing git grep 'pip install' reveals a lot of docs that should be updated to have "--process-dependency-links" and things that should be changed from Flocker[dev] to .[dev]  (I think).

Specifically:
- tox.ini
- CONTRIBUTING.rst
- docs/gettinginvolved/*

Might make more sense as a follow up. Right now I jut went to Slack to look up that line, and I realized it'd be nicer if it was in our official docs.